client/cypress/end-to-end/ballot-comparison-audit-creation.spec.js

```diff
@@ -0,0 +1,370 @@
+
+  it('participating jurisdictions invalid CSV - Admin Email column error', () => {
+    cy.fixture('CSVs/jurisdiction/sample_jurisdiction_filesheet_admin_email_col_error.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'sample_jurisdiction_filesheet_admin_email_col_error.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.get('.Toastify').find('div').find('div').contains('Missing required CSV field "Admin Email"').invoke('text')
+      .then((text)=>{
+        const toastText = text;
+        expect(toastText).to.equal('Missing required CSV field "Admin Email"');
+    })   
+  })
+
+  it('participating jurisdictions invalid CSV - Invalid Email error', () => {
+    cy.fixture('CSVs/jurisdiction/sample_jurisdiction_filesheet_email_ID_error.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'sample_jurisdiction_filesheet_email_ID_error.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.contains("Expected an email address in column Admin Email")   
+  })
+
+  it('participating jurisdictions proper CSV', () => {
+    cy.fixture('CSVs/jurisdiction/sample_jurisdiction_filesheet.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'sample_jurisdiction_filesheet.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.contains("Upload successfully completed")   
+  })
+
+  it('Standardized Contests invalid CSV - Contest name column error', () => {
+    cy.fixture('CSVs/jurisdiction/sample_jurisdiction_filesheet.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'sample_jurisdiction_filesheet.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.contains("Upload successfully completed")   
+
+    cy.fixture('CSVs/contest/sample_standardized_contests_contest_name_col_error.csv').then(fileContent => {
+      cy.get('input[type="file"]').last().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'sample_standardized_contests_contest_name_col_error.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').click()
+    cy.contains("Missing required column: Contest Name.")      
+  })
+
+  it('Standardized Contests invalid CSV - Jurisdiction column error', () => {
```

Comment:
I don't think we need to test all the different file error cases in Cypress. Those are already covered by unit tests. One error case per file type should suffice to convince us that the integration between the frontend and backend is working correctly.